### ISSUE TITLE
cannot clone public repositories

### ISSUE THREAD

**Issue by bjamesvERT** [NOAA-PMEL/GHOrgSync#3]
Script fails to clone public repositories, when no SSH keypair has been set up on GitHub.com

Error: `Permission denied (publickey).`

e.g.:
```$ python2 ghorgsync.py nwfsc-fram repos/nwfsc-fram
Permission denied (publickey).
fatal: The remote end hung up unexpectedly
2018-09-10 11:22:10.462167 :: cannot clone repository sumpis : git@github.com:nwfsc-fram/sumpis.git
```

**Comment by karlmsmith** [comment]
From the error message it appears it was communicating for awhile with GitHub, so I suspect this was a denial of service due to too many requests from an anonymous user in a short period of time.  If anonymous, GitHub greatly reduces the allowed frequency of requests.

**Comment by bjamesvERT** [comment]
it looks like cloning via SSH requires an SSH keypair

per https://help.github.com/articles/which-remote-url-should-i-use/#cloning-with-ssh-urls

**Comment by karlmsmith** [comment]
Thank you for the fix.  Continues to work fine with the SSH keypair.  I will let you close this out as I don't have an easy way of checking that it is working without the SSH keypair.

**Comment by bjamesvERT** [comment]
Issue resolved in #5

(Checked the latest master revision on my system, and clone of public repos works great without an SSH keypair now)
